src/oop/programs_with_interesting_classes/easy/q4/MainTest.java

```diff
@@ -0,0 +1,258 @@
+package oop.programs_with_interesting_classes.easy.q4;
+
+import global.ObjectTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static global.tools.CustomAssertions._assertEquals;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
```

Comment:
Can you please add the label that this is a parsons question to the start of the test as well like we did in other categories please?